python/grass/jupyter/interactivemap.py

```diff
@@ -346,6 +347,89 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def setup_drawing_interface(self):
+        """
+        Allow Users to draw Geometry and Save/Delete them.
+        """
+        import ipywidgets as widgets  # pylint: disable=import-outside-toplevel
+
+        draw_control = self._ipyleaflet.DrawControl()
```

Comment:
I would disable editing and deleting (`DrawControl(edit=False, remove=False)`) to make things simpler.

Reply:
So, aren't we given the option to delete the drawn geometries?

Reply:
Exactly, this should be really simple interaction, no moving or deleting, you can always scratch it and start again. Later, we can think if we want to add this functionality, but for now, stick with the simplest.